README.md

```diff
@@ -25,9 +25,9 @@ We don't have any particular recomendation about hosting providers for tellaweb:
 
 ## NGINX Configuration
 
-for out of the box functionality we added nginx and let's encrypt containers that create a ssl certificate for the domain and serves the site in the :443 port
+To make Tella Web as easy to use out of the box, we added nginx and let's encrypt containers that create a ssl certificate for the domain and serves the site in the :443 port.
```

Comment:
@juandans01 when you write "we added nginx and let's encrypt containers that create a ssl certificate for the domain", do you mean that the nginx and let's encrypt containers are included in the docker container?